filebeat/module/apache/error/test/test.log

```diff
@@ -1,3 +1,4 @@
 [Mon Dec 26 16:22:08 2016] [error] [client 192.168.33.1] File does not exist: /var/www/favicon.ico
 [Mon Dec 26 16:15:55.103786 2016] [core:notice] [pid 11379] AH00094: Command line: '/usr/local/Cellar/httpd24/2.4.23_2/bin/httpd'
 [Fri Sep 09 10:42:29.902022 2011] [core:error] [pid 35708:tid 4328636416] [client 72.15.99.187] File does not exist: /usr/local/apache2/htdocs/favicon.ico
+[Thu Jun 27 06:58:09.169510 2019] [include:warn] [pid 15934] [client 123.123.123.123:12345] AH01374: mod_include: Options +Includes (or IncludesNoExec) wasn't set, INCLUDES filter removed: /test.html
```

Comment:
We will also need to add the expected result of this entry to the `test.log-expected.json` file. You can automatically generate it running the tests with `GENERATE=1` from the filebeat directory:
```
GENERATE=1 INTEGRATION_TESTS=1 BEAT_STRICT_PERMS=false TESTING_FILEBEAT_MODULES=apache nosetests tests/system/test_modules.py
```

As doing the setup for these tests is a bit time-consuming sometimes, I have generated it:
```
    {
        "@timestamp": "2019-06-27T06:58:09.169Z",
        "apache.error.module": "include",
        "event.dataset": "apache.error",
        "event.module": "apache",
        "fileset.name": "error",
        "input.type": "log",
        "log.level": "warn",
        "log.offset": 384,
        "message": "AH01374: mod_include: Options +Includes (or IncludesNoExec) wasn't set, INCLUDES filter removed: /test.html",
        "process.pid": 15934,
        "service.type": "apache",
        "source.address": "123.123.123.123",
        "source.geo.city_name": "Beijing",
        "source.geo.continent_name": "Asia",
        "source.geo.country_iso_code": "CN",
        "source.geo.location.lat": 39.9288,
        "source.geo.location.lon": 116.3889,
        "source.geo.region_iso_code": "CN-BJ",
        "source.geo.region_name": "Beijing",
        "source.ip": "123.123.123.123",
        "source.port": "12345"
    }
```